src/Illuminate/Queue/Events/WorkerStopping.php

```diff
@@ -15,5 +15,6 @@ public function __construct(
         public $status = 0,
         public $workerOptions = null
     ) {
+        $this->status = $status;
```

Comment:
Did you intend this change or is that a leftover from a previous iteration of this PR?

Reply:
[Good catch 👍🏻 It's a leftover, I missed](https://github.com/laravel/framework/pull/53961#discussion_r1891250982)